public/components/notebooks/components/note_table.tsx

```diff
@@ -114,7 +124,10 @@ export function NoteTable(props: NoteTableProps) {
     setModalLayout(
       getCustomModal(
         onCreate,
-        closeModal,
+        () => {
+          closeModal()
+          history.push('/notebooks')
```

Comment:
Should this be the `/notebooks` or the create workflow originator page?

Reply:
Good catch - I missed this as I was mainly testing it from the notebooks page itself, not the connection from other pages. Fixed with h`history.goback()` in [cc0763c](https://github.com/opensearch-project/dashboards-observability/pull/152/commits/cc0763cb6612d999418849812427d33a7ca931f8)